lib/mix/lib/mix/tasks/test.ex

```diff
@@ -569,6 +575,22 @@ defmodule Mix.Tasks.Test do
     end
   end
 
+  defp normalize_max_failures(opts) do
+    case Keyword.fetch(opts, :max_failures) do
```

Comment:
😅 I think you missed the comment related to the `:error`. I think this code will raise `** (CaseClauseError) no case clause matching: :error`

Reply:
I think it should work now 👍